common/network-common/src/test/java/org/apache/spark/network/server/OneForOneStreamManagerSuite.java

```diff
@@ -30,22 +31,47 @@
 public class OneForOneStreamManagerSuite {
 
   @Test
-  public void managedBuffersAreFeedWhenConnectionIsClosed() throws Exception {
+  public void testMissingChunk() {
     OneForOneStreamManager manager = new OneForOneStreamManager();
     List<ManagedBuffer> buffers = new ArrayList<>();
     TestManagedBuffer buffer1 = Mockito.spy(new TestManagedBuffer(10));
     TestManagedBuffer buffer2 = Mockito.spy(new TestManagedBuffer(20));
+    TestManagedBuffer buffer3 = Mockito.spy(new TestManagedBuffer(20));
+
     buffers.add(buffer1);
+    buffers.add(null);
```

Comment:
please add a comment about `null`s here are to simulate a file which goes missing before its read, just as a defensive measure.